docs/hyacinth-2.5/uuid-identifiers.md

```diff
@@ -0,0 +1,10 @@
+# UUID Identifiers
+
+Hyacinth switched to using UUID (v4) identifiers in version 2.5, previously using Fedora 3 PIDs (e.g. abc:123).
```

Comment:
Can this be "internal identifiers"?

Reply:
Good suggestion.  Just changed.  Thanks!